docs/user-guide/filters.md

```diff
@@ -0,0 +1,110 @@
+
+  1. **Messages are directed and unique, values are persistent.** - Because HELICS values are used to represent physical reality, they are available to any subscribing federate at any time. If the publishing federate only updates the value, say, once every minute, any subscribing federates that are granted a time during that minute window will all receive the same value regardless of when they requested it. 
+
+     HELICS messages, though, are much more like other kinds of real-world messages in that they are directed and unique. Messages are sent by a federate from one endpoint to another endpoint in the federation (presumably the receiving endpoint is owned by another federate but this doesn't have to be the case). Internal to HELICS, each message has a unique identifier and can be thought to travel between a generic communication system between the source and destination endpoints.
+  
+  2. **Messages can be filtered, values cannot.** - By creating a generic communication system between two endpoints, HELICS has the ability to model simple communication system effects on messages traveling through said network. These effects are called "filters" and are associated with the HELICS core (which in turn manages the federate's endpoints) embedded with the federate in question. Typical filtering actions might be delaying the transmission of the message or randomly dropping a certain percentage of the received messages. Filters can also be defined to operate on messages being sent ("source filters") and/or messages being received ("destination filters").
+
+     Because HELICS values do not pass through this generic network, they cannot be operated on by filters. Since HELICS values are used to represent physics of the system not the control and coordination of it, it is appropriate that filters not be available to modify them. It is entirely possible to use HELICS value federates to send control signals to other federates; co-simulations can and have been made to work in such ways. Doing so, though, cuts out the possibility of using filters and, as we'll see, the easy integration of communication system simulators.
+  
+The figure below is an example of a representation of the message topology of a generic co-simulation federation composed entirely of message federates. Source and destination filters have been implemented (indicated by the blue endpoints), each showing a different built-in HELICS filter function. 
+  
+  * As a reminder, a single endpoint can be used to both send and receive messages (as shown by Federate 4) but a single filter will only act on messages sent or messages received. Both a source filter and a destination filter can be set up on a single endpoint.
```

Comment:
With HELICS 2.0 filters can act on as many endpoints as desired, sending or receiving.  I wasn't sure how to specify that in this list.    Filters have destination targets and source targets.    There are no longer separate filter types for source and destination.

Reply:
@trevorhardy  did you want me to try to update this section at the risk of it being a little divergent in style?

Reply:
I think that would be fine. I would prefer that the content be technically accurate over stylistically consistent. Once you're done I can go back through the stuff you changed and see if I can bring the style back in line.